freqtrade/freqai/data_drawer.py

```diff
@@ -293,6 +294,15 @@ def append_model_predictions(self, pair: str, predictions: DataFrame,
             for return_str in rets:
                 df[return_str].iloc[-1] = rets[return_str]
 
+        # this logic carries users between version without needing to 
+        # change their identifier
+        if 'close_price' not in df.columns:
+            df['close_price'] = 0
+            df['date_pred'] = 0
```

Comment:
I think this field should have a different type.
Right now - we'd create an empty column of type int (most likely np.int64).

the actual date column is however of type pd.Timestamp ... so the below 2 lines will cause the type of this field to be object (it's mixed type now) ...

Reply:
good point